schemas/oic.r.pstat-Update.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://www.openconnectivity.org/ocf-apis/security/schemas/oic.r.pstat-Update.json#",
+  "title": "Device Provisioning Status information",
+  "definitions": {
+    "oic.r.pstat": {
+      "type": "object",
+      "properties": {
+        "dos": {
+          "allOf": [
+            {
+              "type": "object",
+              "description": "Device on-boarding state"
+            },
+            {
+              "$ref": "oic.sec.dostype.json#/definitions/oic.sec.dostype"
+            }
+          ]
+        },
+        "tm": {
+          "allOf": [
+            {
+              "description": "Target device provisioning mode"
+            },
+            {
+              "$ref": "oic.sec.dpmtype.json#/definitions/oic.sec.dpmtype/properties/bitmask"
+            }
+          ]
+        },
+        "om": {
+          "allOf": [
+            {
+              "description": "Current operational mode"
+            },
+            {
+              "$ref": "oic.sec.pomtype.json#/definitions/oic.sec.pomtype/properties/bitmask"
+            }
+          ]
+        },
+        "rowneruuid": {
+          "allOf": [
+            {
+              "description": "The UUID formatted identity of the Resource owner"
+            },
+            {
+              "$ref": "../../core/schemas/oic.types-schema.json#/definitions/uuid"
+            }
+          ]
+        }
+      }
+    }
+  },
+  "type": "object",
+  "allOf": [
+    { "$ref": "../../core/schemas/oic.core-schema.json#/definitions/oic.core" },
+    { "$ref": "#/definitions/oic.r.pstat" }
+  ],
+  "required": [ "dos", "tm", "om", "rowneruuid" ]
```

Comment:
"required" Properties should be empty; any combination is valid

Reply:
Agreed.